server/test/utils.ts

```diff
@@ -0,0 +1,94 @@
+import { rand, randEmail, randNumber, randUserName, randQuote, randBrand } from '@ngneat/falso';
+
+import { User } from 'server/models';
+
+// user interface
```

Comment:
in this case i think they can/should be optional, since it will fall back to random values if not set.  e.g. "i want a user with this specific email but i dont care about the other values"

Reply:
but in any case you can use the `Partial` type for this: https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype

Reply:
> in this case i think they can/should be optional, since it will fall back to random values if not set. e.g. "i want a user with this specific email but i dont care about the other values"

Ahh, you're right... I missed that it was being used for incoming params and confused it with `userObject`, which isn't explicitly given a type. 👍🏼